src/vcpkg/registries.cpp

```diff
@@ -158,11 +158,11 @@ namespace
         }
 
         View<Version> get_port_versions() const override { return {&version, 1}; }
-        ExpectedS<Path> get_path_to_version(const Version& v) const override
+        ExpectedS<PathAndLocation> get_version(const Version& v) const override
         {
             if (v == version)
             {
-                return root;
+                return PathAndLocation{root, "git+https://github.com/Microsoft/vcpkg#ports/" + name};
```

Comment:
I wasn't sure whether to version this or not. This case is only for non-baseline users, and I wanted to avoid doing anything expensive (like git operations or loading additional data).

Note that the actual version text is recorded separately. This is just intended to represent a best-effort location to access the package.